[2022-09-16 13:07:56,258] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: preprocessing_data.combine_data manual__2022-09-16T13:06:13.635440+00:00 [queued]>
[2022-09-16 13:07:56,264] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: preprocessing_data.combine_data manual__2022-09-16T13:06:13.635440+00:00 [queued]>
[2022-09-16 13:07:56,264] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-16 13:07:56,264] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-09-16 13:07:56,264] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-16 13:07:56,280] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): combine_data> on 2022-09-16 13:06:13.635440+00:00
[2022-09-16 13:07:56,287] {standard_task_runner.py:52} INFO - Started process 48498 to run task
[2022-09-16 13:07:56,291] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'preprocessing_data', 'combine_data', 'manual__2022-09-16T13:06:13.635440+00:00', '--job-id', '351', '--raw', '--subdir', 'DAGS_FOLDER/model_dag.py', '--cfg-path', '/tmp/tmpslfjdahd', '--error-file', '/tmp/tmp59peysyy']
[2022-09-16 13:07:56,292] {standard_task_runner.py:80} INFO - Job 351: Subtask combine_data
[2022-09-16 13:07:56,347] {task_command.py:371} INFO - Running <TaskInstance: preprocessing_data.combine_data manual__2022-09-16T13:06:13.635440+00:00 [running]> on host rc1b-dataproc-m-xzxwmqcudfo0foh0.mdb.yandexcloud.net
[2022-09-16 13:07:56,397] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=preprocessing_data
AIRFLOW_CTX_TASK_ID=combine_data
AIRFLOW_CTX_EXECUTION_DATE=2022-09-16T13:06:13.635440+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-16T13:06:13.635440+00:00
[2022-09-16 13:07:56,399] {client.py:192} INFO - Instantiated <Client(url='http://rc1b-dataproc-m-xzxwmqcudfo0foh0.mdb.yandexcloud.net:9870')>.
[2022-09-16 13:08:08,911] {client.py:305} INFO - Fetching content summary for '/user/airflow/input_files'.
[2022-09-16 13:08:29,400] {python.py:173} INFO - Done. Returned value was: None
[2022-09-16 13:08:29,409] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=preprocessing_data, task_id=combine_data, execution_date=20220916T130613, start_date=20220916T130756, end_date=20220916T130829
[2022-09-16 13:08:29,454] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-16 13:08:29,479] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
